Gemfile

```diff
@@ -28,7 +28,11 @@ group :backend, :frontend, :core, :api do
   gem 'database_cleaner', '~> 1.3', require: false
   gem 'factory_bot_rails', '~> 4.8', require: false
   gem 'rspec-activemodel-mocks', '~> 1.1', require: false
-  gem 'rspec-rails', '~> 3.7', require: false
+  gem 'rspec-rails', github: 'rspec/rspec-rails', branch: '4-0-dev', require: false
+  gem 'rspec-core', github: 'rspec/rspec-core', require: false
+  gem 'rspec-mocks', github: 'rspec/rspec-mocks', require: false
+  gem 'rspec-support', github: 'rspec/rspec-support', require: false
+  gem 'rspec-expectations', github: 'rspec/rspec-expectations', require: false
```

Comment:
You can just use `gem 'rspec-rails', '>= 4.0.0.beta2'` instead.

Reply:
@tvdeyen the betas are not good unfortunately, they don't provide full support for Rails 6. We need to wait for 4.0.0 stable. 😞

Reply:
What exactly is the issue? I have good results with the betas in Alchemy.

Reply:
@tvdeyen I think you're right, not sure what kind of problem I experienced initially but it doesn't seem to happen anymore. It was probably not related to RSpec.